patches/api/0175-Fix-Spigot-annotation-mistakes.patch

```diff
@@ -96,6 +105,19 @@ index 2ff1b1308571d8f8056d3359e8a8ba4a589c3726..e669ad8ecd182c6899c7820414e6ee1f
      @NotNull
      public int[] getExpLevelCostsOffered() {
          int[] levelOffers = new int[offers.length];
+@@ -85,8 +86,10 @@ public class PrepareItemEnchantEvent extends InventoryEvent implements Cancellab
+      *
+      * @return list of available enchantment offers
+      */
+-    @NotNull
+-    public EnchantmentOffer[] getOffers() {
++    // Paper start - offers can contain null values
++    // @NotNull
++    public @org.jetbrains.annotations.Nullable EnchantmentOffer @NotNull [] getOffers() {
++    // Paper end
```

Comment:
just put the paper comment at the end of the line. no need to leave the old commented-out annotation or have start/end comments.

Reply:
This has been fixed by the way